cypress/e2e/fetch-status.cy.js

```diff
@@ -0,0 +1,83 @@
+describe("Fetch Status", () => {
+
+    it("Fetch data with no authenticated user", () => {
+
+        cy.visit("/");
+
+        // Go immediately to query
+        cy.contains("Mixed book query").click();
+
+        // Open the sources and verify
+        cy.get('[aria-label="Sources info"]').click();
+        cy.get('[aria-label="Verify source"]').each(($button) => {
```

Comment:
No need to click the verify source buttons here - that is a different test case (verify-source.cy.js)